### PR TITLE
Lidar page adjust, SM numbers fix

### LINKED ISSUE #4: Beta QC-StratMap Page
Can we add a thumbnail link to the  Lidar status map on the Stratmap landing page?

### LINKED ISSUE #114: stratmap orthoimagery & elevation-lidar pages ---> black font over image
https://tnris.org/stratmap/orthoimagery/

the black font on top of the image below on the stratmap orthoimagery page is difficult if not impossible to read. i think it might look better white with a black shadow. the stratmap elevation-lidar page has the same issue - https://tnris.org/stratmap/elevation-lidar/.

Screenshot from 2019-11-21 07-46-48

### PR DESCRIPTION
**Pull request by christoball**
closes #114, closes #4

### CODE REVIEW

**Comment by christoball** [comment]
also closes #113

**Review by adambreznicky — CHANGES_REQUESTED** [review]
Selection_016

sass changes have effected the main /stratmap by the numbers section so the text is not legible

**Comment by adambreznicky** [comment]
i'm not sure how the changes close #4 ? there is no new lidar status map thumbnail link on the stratmap landing page...
i see changes to the organization of the lidar page, but the content is the same with only an exception of an anchor text link to stratmap contracts.

**Review by adambreznicky — CHANGES_REQUESTED** [review]
i'm not sure how the changes close #4 ? there is no new lidar status map thumbnail link on the stratmap landing page...
i see changes to the organization of the lidar page, but the content is the same with only an exception of an anchor text link to stratmap contracts.

File: content/markdown/stratmap/elevation-lidar.md [hunk 1]
```diff
@@ -3,34 +3,35 @@ title: Elevation – Lidar
...
-    </div>
-    <div class="col-md-4">
-      <h3>Lidar Specifications</h3>
-      <a class="btn btn-lg btn-success" href="https://tnris-org-static.s3.amazonaws.com/documents/lidar_spec.pdf"><i class="glyphicon glyphicon-document"></i> Download Lidar Specifications</a>
-      <h3>Download Lidar</h3>
-      <a class="btn btn-lg btn-tnris" href="https://data.tnris.org"><img style="width: 20px; margin-bottom: 0 !important;" src="https://tnris-org-static.s3.amazonaws.com/images/baseline_view_comfy_white_36dp.png"> Launch DataHub</a>
-    </div>
-  </div>
+<div class="row">
+<div class="col-md-8">
```
Comment (adambreznicky) [inline content/markdown/stratmap/elevation-lidar.md]: these nested tags should be indented

File: content/markdown/stratmap/elevation-lidar.md [hunk 2]
```diff
@@ -3,34 +3,35 @@ title: Elevation – Lidar
...
+<p class="lead"><strong>Lidar - Light Detection and Ranging,</strong> is a remote sensing technique that utilizes light in the form of a rapidly pulsed laser to measure return distances from the Earth captured by a sensor at the source of the pulse. These combined pulse return measurements with additional spatial and temporal data recorded by the acquisition system (airborne or terrestrial) produce a three-dimensional (3-D), detailed representation of the shape of the Earth illuminating its surface characteristics.</p>
+<p class="lead"> TNRIS acquires lidar data through partnerships with other federal and state agencies through the <a href="/stratmap/stratmap-contracts"><strong>StratMap Contract</strong></a>, which operates through the <a href="https://dir.texas.gov/">Texas Department of Information Resources (TxDIR)</a>.
+<h2>TNRIS Lidar Coverage</h2>
```
Comment (adambreznicky) [inline content/markdown/stratmap/elevation-lidar.md]: numerous nested tags in this file should be properly indented based on parent tags

**Comment by christoball** [comment]
@ThorTheDestroyer and this is the approach we agreed on. We moved the Lidar status map up in the page and we will revisit the front page at a future date.

**Comment by christoball** [comment]
@adambreznicky @ThorTheDestroyer we rearranged the stratmap page and will revisit the front page at a later date.

**Comment by adambreznicky** [comment]
> @ThorTheDestroyer and this is the approach we agreed on. We moved the Lidar status map up in the page and we will revisit the front page at a future date.

right on, sounds good. i agree, the lidar status map is probably more important than the side-by-side slider. glad that just moving it is enough to satisfy the request. maybe next time throw the context/conclusion/decision of y'alls chat into the issue? without it i was unaware of the change in plan during review. super thanks

**Comment by christoball** [comment]
@adambreznicky I'll be sure to make it clear if there are major changes in the approach to resolving stuff on future issues, that makes sense especially to help review.

**Comment by christoball** [comment]
@adambreznicky all the different number sections and their colors should be correct now with the latest commit

**Comment by adambreznicky** [comment]
> @adambreznicky all the different number sections and their colors should be correct now with the latest commit

looks great!

**Pull request merged**